Lets Do This/Controllers/Base/LDTWebViewController.m

```diff
@@ -53,5 +59,32 @@ - (void)viewDidAppear:(BOOL)animated {
     [[GAI sharedInstance] trackScreenView:self.screenName];
 }
 
+- (void)viewWillDisappear:(BOOL)animated {
+    [super viewWillDisappear:animated];
+    
+    self.navigationController.toolbarHidden = YES;
+}
+
+#pragma mark - IBActions
+
+- (IBAction)downloadButtonTapped:(id)sender {
+    UIBarButtonItem *item = (UIBarButtonItem *)self.toolbarItems[0];
+    NSURLRequest *request = [NSURLRequest requestWithURL:self.webViewURL];
+    NSString *documentDir = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) firstObject];
+    NSString *filePath = [documentDir stringByAppendingPathComponent:self.webViewURL.lastPathComponent];
+    [NSURLConnection sendAsynchronousRequest:request queue:[NSOperationQueue currentQueue] completionHandler:^(NSURLResponse *response, NSData *data, NSError *error) {
+        if (error) {
+            // @todo LDTMessage
+            NSLog(@"Download Error: %@", error.description);
+        }
+        if (data) {
+            [data writeToFile:filePath atomically:YES];
+            NSURL *fileURL = [NSURL fileURLWithPath:filePath];
+            UIDocumentInteractionController *documentInteractionController = [UIDocumentInteractionController interactionControllerWithURL:fileURL];
```

Comment:
Getting this crash on Doggie DIY. [Sounds like](http://stackoverflow.com/a/2926599/1470725) we should try the init in `viewDidLoad` instead of on demand:

> anOpenURL: failed for URL: "about:blank" - error: "This app is not allowed to query for scheme about"
objc[699]: UIPDFPageRenderOperation object 0x14960b810 overreleased while already deallocating; break on objc_overrelease_during_dealloc_error to debug
2016-05-25 12:41:27.195 DoSomething[699:270196] *** Assertion failure in -[_UIOpenWithAppActivity performActivity], /BuildRoot/Library/Caches/com.apple.xbs/Sources/UIKit/UIKit-3512.60.12/UIDocumentInteractionController.m:408
2016-05-25 12:41:27.197 DoSomething[699:270196] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'UIDocumentInteractionController has gone away prematurely!'
*** First throw call stack:
(0x18181edb0 0x180e83f80 0x18181ec80 0x1821a4154 0x18701ecf8 0x1872521b4 0x186cd9944 0x186cdcb3c 0x186aad114 0x1869b812c 0x1869b7c28 0x1869b7ab0 0x1843119a0 0x100a01a3c 0x100a074e4 0x1817d4d50 0x1817d2bb8 0x1816fcc50 0x182fe4088 0x1869e6088 0x1000bdc84 0x18129a8b8)
libc++abi.dylib: terminating with uncaught exception of type NSException